pkg/reading/service.go

```diff
@@ -0,0 +1,62 @@
+package reading
+
+import (
+	"errors"
+	"fmt"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/helpers"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/http/rest"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/service"
+	"net/http"
+)
+
+type ServiceReadIssue struct {
+	issueId   string
+	fieldKey  string
+	fieldName string
+}
+
+func (s *ServiceReadIssue) InitJiraAPI(params configuration.JiraAPIResourceParameters) (rest.JiraAPI, error) {
+	s.issueId = params.ActiveIssue
+	s.fieldName = *params.CustomFieldName
+
+	return service.PreInitJiraAPI(s, params, http.MethodGet)
+}
+
+func (s *ServiceReadIssue) GetResults() map[string]string {
+	var m = make(map[string]string)
+	m[helpers.ReadingFieldKey] = s.fieldKey
+	return m
+}
+
+func (s *ServiceReadIssue) SetResultsFromPrevious(result map[string]string) {
+}
+
+func (s *ServiceReadIssue) GetEndpoint(url string) string {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ServiceReadIssue.GetEndpoint should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)